schedulingApp/unit-tests/test_accounts.py

```diff
@@ -173,62 +175,57 @@ def test_noChangePermission(self):
 
     # To email used in another account
     def test_toConflictingEmail(self):
-        a = User("Rob", "Adams", "arob@uwm.edu", "Otherplace Rd 234", "4567891234", "ta")
+        user1 = User.objects.create_user(username="Rob Adams", email="arob@uwm.edu", password="pass")
+        profile1 = Profile(user=user1, phoneNumber="4567891234", homeAddress="Otherplace Rd 234", permission=Profile.TA)
 
-        a.setEmailAddress("srobert@uwm.edu")
-        self.assertFalse(a.emailAddress == "srobert@uwm.edu",
+        profile1.setEmailAddress("srobert@uwm.edu")
+        self.assertFalse(profile1.getEmailAddress() == "srobert@uwm.edu",
                          msg="Email should not match any other accounts")
 
     def test_toConflictingPhoneNumbers(self):
-        a = User("Rob", "Adams", "arob@uwm.edu", "Otherplace Rd 234", "4567891234", "ta")
+        user1 = User.objects.create_user(username="Rob Adams", email="arob@uwm.edu", password="pass")
+        profile1 = Profile(user=user1, phoneNumber="4567891234", homeAddress="Otherplace Rd 234", permission=Profile.TA)
 
-        a.setPhoneNumber("2345678901")
-        self.assertFalse(a.phoneNumber == "2345678901",
+        profile1.setPhoneNumber("2345678901")
+        self.assertFalse(profile1.getPhoneNumber() == "2345678901",
                          msg="Phone number should not match any other accounts")
 
 
 class DeleteAccount(TestCase):
     theUser = None
 
     def test_accountNotFound(self):
-        user = User.objects.get(self.theUser)
+        user = Profile.objects.get(self.theUser)
         self.assertRaises(NameError, user.delete(),
                           msg="Trying to delete nonexistent user should throw NameError")
 
     def setUp(self):
-        theUser = User("Robert", "Smith", "dogs123", "srobert@uwm.edu", "Someplace Ave 123", "2345678901", "admin")
-        theUser.firstName = "Robert"
-        theUser.lastName = "Smith"
-        theUser.emailAddress = "srobert@uwm.edu"
-        theUser.homeAddress = "Someplace Ave 123"
-        theUser.phoneNumber = "2345678901"
-        theUser.permission = "admin"
+        user = User.objects.create_user(username="Robert Smith", email="srobert@uwm.edu", password="pass")
+        profile = Profile(user=user, phoneNumber="2345678901", homeAddress="Someplace Ave 123", permission=Profile.TA)
```

Comment:
use self.user = ... and self.profile = ...
also define those fields in the class itself